opentech/apply/blocks.py

```diff
@@ -21,6 +21,17 @@ def target_model(self):
 
 
 class CategoryQuestionBlock(FormFieldBlock):
+    # Overwrite field label and help text so we can defer to the category
+    # as required
+    field_label = CharBlock(
+        label=_('Label'),
+        required=False,
+        help_text=_('Leave blank to use the default on the category'),
```

Comment:
'Leave blank to use the default Category label' has a nicer sound to it. And it feels category should be uppercase